src/components/pluginManager.js

```diff
@@ -52,41 +53,36 @@ import globalize from 'globalize';
 
         loadPlugin(pluginSpec) {
             if (typeof pluginSpec === 'string') {
-                console.debug('Loading plugin (via deprecated requirejs method): ' + pluginSpec);
-
-                return new Promise((resolve, reject) => {
-                    require([pluginSpec], (pluginFactory) => {
-                        const plugin = pluginFactory.default ? new pluginFactory.default() : new pluginFactory();
+                console.debug('Loading plugin (via dynamic import): ' + pluginSpec);
 
-                        // See if it's already installed
-                        const existing = this.pluginsList.filter(function (p) {
-                            return p.id === plugin.id;
-                        })[0];
+                import(/* webpackChunkName: "[request]" */ `../plugins/${pluginSpec}`).then((plugin) => {
```

Comment:
removed all requirejs related code, since it's not used any more (at least i think so).

Makes sense to separate each plugin, since requirements may change between clients, so modules seems ok.

Reply:
About `plugin`. Looking at this implementation https://github.com/jellyfin/jellyfin-android/blob/6c89ce1be8f53f2918a885db2b8819f5ecbc58dd/app/src/main/assets/native/exoplayer.js#L4
I believe that the lack of a `new plugin` breaks it.